deploy/infrastructure/common/github_actions.tf

```diff
@@ -6,7 +6,8 @@ resource "aws_iam_openid_connect_provider" "github" {
   ]
 
   thumbprint_list = [
-    "6938fd4d98bab03faadb97b34396831e3780aea1"
+    "6938fd4d98bab03faadb97b34396831e3780aea1",
+    "a3b59e5fe884ee1f34d98eef858e3fb662ac104a"
```

Comment:
Should this be `1c58a3a8518e8759bf075b76b750d4f2df264fcd` ?

Reply:
Maybe.  I grabbed "a3b59e5fe884ee1f34d98eef858e3fb662ac104a" from the github.com certificate that I saw, and I tested that using it fixed the problem. If there are other valid github certs then we should include their fingerprints as well.